spread.yaml

```diff
@@ -158,10 +158,6 @@ backends:
             - opensuse-tumbleweed-64:
                   workers: 6
                   manual: true
-            - centos-8-64:
```

Comment:
to be clear this is "CentOS 8" and **not** "Centos Stream 8" correct? The latter seems to have an EOL date in 2024 so it doesn't make sense to drop that yet but the former indeed went EOL at the end of 2021.

Reply:
Yes, that's right